tools/packaging/packaging.psm1

```diff
@@ -8,7 +8,7 @@ $packagingStrings = Import-PowerShellDataFile "$PSScriptRoot\packaging.strings.p
 Import-Module "$PSScriptRoot\..\Xml" -ErrorAction Stop -Force
 $DebianDistributions = @("ubuntu.16.04", "ubuntu.18.04", "ubuntu.20.04", "debian.9", "debian.10", "debian.11")
 $RedhatDistributions = @("rhel.7","centos.8")
-$script:netCoreRuntime = 'net5.0'
+$script:netCoreRuntime = 'net6.0'
```

Comment:
same here, are the changes here done by the script or manually? I'd like to know what manual actions are left.

Reply:
fixed here and many other places,

Reply:
hmm, good news is that we don't have to do this part of work again until bump to net7.0 :)